### ISSUE TITLE
2.2.8: build fails

### ISSUE THREAD

**Issue by kloczek** [power-ras/libvpd#4]
gcc 11.0.1
```
[tkloczko@barrel libvpd-2.2.8]$ make
/bin/sh ./libtool  --tag=CXX   --mode=compile /usr/bin/g++ -DHAVE_CONFIG_H -I. -I./config  -I./src/ -Wall -fstack-protector-all -fPIE -Wstack-protector  -DDEST_DIR='"/usr"' -O2 -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fdata-sections -ffunction-sections -flto=auto -flto-partition=none -c -o src/vpdretriever.lo src/vpdretriever.cpp
libtool: compile:  /usr/bin/g++ -DHAVE_CONFIG_H -I. -I./config -I./src/ -Wall -fstack-protector-all -Wstack-protector -DDEST_DIR=\"/usr\" -O2 -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fdata-sections -ffunction-sections -flto=auto -flto-partition=none -c src/vpdretriever.cpp  -fPIC -DPIC -o src/.libs/vpdretriever.o
In file included from src/vpdretriever.cpp:25:
./src/libvpd-2/vpdretriever.hpp:62:33: error: ISO C++17 does not allow dynamic exception specifications
   62 |                                 throw( VpdException& );
      |                                 ^~~~~
./src/libvpd-2/vpdretriever.hpp:74:33: error: ISO C++17 does not allow dynamic exception specifications
   74 |                                 throw( VpdException& );
      |                                 ^~~~~
src/vpdretriever.cpp:50:37: error: ISO C++17 does not allow dynamic exception specifications
   50 |                 string dbFileName ) throw( VpdException& )
      |                                     ^~~~~
src/vpdretriever.cpp:62:39: error: ISO C++17 does not allow dynamic exception specifications
   62 |         VpdRetriever::VpdRetriever( ) throw( VpdException& )
      |                                       ^~~~~
make: *** [Makefile:671: src/vpdretriever.lo] Error 1
```

**Comment by kamalesh-babulal** [comment]
We need to change the lsvpd code too. I will try and fix at both the packages.

**Comment by kloczek** [comment]
If you will have patch for that please let me know. I would be glad to test that as well :)

**Comment by kamalesh-babulal** [comment]
> If you will have patch for that please let me know. I would be glad to test that as well :)

Thank you, I was out for a week.

**Comment by kloczek** [comment]
Just tested https://github.com/power-ras/libvpd/pull/5 and looks like wit that patch libvpd builds correctly.

Thx :)

**Comment by hegdevasant** [comment]
Thanks for the verification @kloczek .

-Vasant

**Comment by hegdevasant** [comment]
Merged to master as 83ccb994e3.

-Vasant
